filebeat/tests/system/test_prospector.py

```diff
@@ -103,3 +103,56 @@ def test_rotating_ignore_older_larger_write_rate(self):
             max_timeout=15)
 
         proc.kill_and_wait()
+
+    def test_rotating_ignore_older_low_write_rate(self):
+        self.render_config_template(
+            path=os.path.abspath(self.working_dir) + "/log/*",
+            ignoreOlder="1s",
+            scan_frequency="0.1s",
+        )
+
+        os.mkdir(self.working_dir + "/log/")
+        testfile = self.working_dir + "/log/test.log"
+
+        proc = self.start_filebeat(debug_selectors=['*'])
+
+        # wait for first  "Start next scan" log message
+        self.wait_until(
+            lambda: self.log_contains(
+                "Start next scan"),
+            max_timeout=10)
+
+        lines = 0
+
+        # write first line
+        lines += 1
+        with open(testfile, 'a') as file:
+            file.write("Line {}\n".format(lines))
+
+        # wait for log to be read
+        self.wait_until(
+            lambda: self.output_has(lines=lines),
+            max_timeout=15)
+
+        # log rotate
+        os.rename(testfile, testfile + ".1")
+        open(testfile, 'w').close()
+
+        # wait for file to be closed due to ignore_older
+        self.wait_until(
+            lambda: self.log_contains(
+              "Error: Stop harvesting as file is older then ignore_older: {};".format(os.path.abspath(testfile))),
+            max_timeout=10)
+        time.sleep(2)
```

Comment:
This timeout is actually what causes the test to fail - I think because it allows the second ignore older message for the same file to come through, after which things get into a non working state